habitat-lab/habitat/core/utils.py

```diff
@@ -135,14 +137,15 @@ def default(self, obj):
 
 
 class DatasetFloatJSONEncoder(DatasetJSONEncoder):
-    r"""JSON Encoder that sets a float precision for a space saving purpose and
+    """JSON Encoder that sets a float precision for a space saving purpose and
     encodes ndarray and quaternion. The encoder is compatible with JSON
     version 2.0.9.
     """
 
-    # Overriding method to inject own `_repr` function for floats with needed
-    # precision.
     def iterencode(self, o, _one_shot=False):
+        """
+        Overriding method to inject own `_repr` function for floats with needed precision.
```

Comment:
Missing parameter descriptons for "o" and "_one_shot".  Not sure if this also overriding something in "DatasetJSONEncode"?  Is parameter "o" the same as the "o" in the floatstr function below?

:param o: the float to convert?